spec/abstractops.html

```diff
@@ -1241,9 +1241,8 @@ <h1>
     <emu-note>The value of ! ToIntegerOrInfinity(*undefined*) is 0.</emu-note>
     <emu-alg>
       1. Let _parseResult_ be ~empty~.
-      1. For each nonterminal _goal_ of &laquo; |TemporalDateTimeString|, |TemporalInstantString|, |TemporalMonthDayString|, |TemporalTimeString|, |TemporalYearMonthString|, |TemporalZonedDateTimeString| &raquo;, do
-        1. If _parseResult_ is not a Parse Node, set _parseResult_ to ParseText(StringToCodePoints(_isoString_), _goal_).
-      1. Assert: _parseResult_ is a Parse Node.
+      1. Set _parseResult_ to ParseText(StringToCodePoints(_isoString_), |ISODateTimeString|).
```

Comment:
Assigning `empty` in the first step is redundant, you can remove that and change this to:

```suggestion
      1. Let _parseResult_ be ParseText(StringToCodePoints(_isoString_), |ISODateTimeString|).
```